package.json

```diff
@@ -370,8 +370,8 @@
     "@typescript-eslint/eslint-plugin": "4.22.0",
     "@typescript-eslint/experimental-utils": "4.22.0",
     "@typescript-eslint/parser": "4.22.0",
-    "@types/jest-axe/axe-core": "4.4.3",
-    "jest-axe/axe-core": "4.4.3",
+    "@types/jest-axe/axe-core": "4.7.2",
+    "jest-axe/axe-core": "4.7.2",
```

Comment:
This resolution is no longer necessary as jest-axe specifies axe-core 4.7.2

```suggestion
```

Reply:
Ok